packages/sandbox/src/App.tsx

```diff
@@ -1,180 +1,268 @@
-import React from 'react';
+import React, {Component, FC} from 'react';
```

Comment:
first of all, We really appreciate this update, thanks for dedicated your time. The PR looks good to me but a I will ask you a last favor 🙊 . Can you please order the components alphabetically 🙏🏾  ? Is to keep the same convention of the folder `packages/css/src/atoms`

Reply:
done :D

Reply:
Thanks!